extensions/redis-client/runtime/src/main/java/io/quarkus/redis/client/runtime/RedisConfig.java

```diff
@@ -150,5 +158,30 @@
          */
         @ConfigItem(defaultValue = "32")
         public int maxNestedArrays;
+
+        /**
+         * The number of reconnection attempts when a pooled connection cannot be established on first try.
+         */
+        @ConfigItem(defaultValue = "0")
+        public int reconnectAttempts = 0;
+
+        /**
+         * The interval between reconnection attempts when a pooled connection cannot be established on first try.
+         */
+        @ConfigItem(defaultValue = "PT1S")
```

Comment:
```suggestion
        @ConfigItem(defaultValue = "1")
```

Reply:
Is this correct?
This field type is Duration, should the format not be a text string such as PnDTnHnMn.nS?